app/src/main/res/layout/list_item_message_received.xml

```diff
@@ -25,22 +26,39 @@
         android:layout_marginLeft="@dimen/meshim_margin_small"
         android:layout_marginStart="@dimen/meshim_margin_small"
         android:background="@drawable/chat_bubble_rec"
-        android:padding="@dimen/meshim_margin_small"
         android:elevation="@dimen/elevation_card"
+        android:padding="@dimen/meshim_margin_small"
         android:textColor="@color/white"
-        app:layout_constraintStart_toEndOf="@id/image_message_profile"
         app:layout_constraintLeft_toRightOf="@+id/image_message_profile"
-        app:layout_constraintRight_toRightOf="parent"
```

Comment:
Does removing these have any effect on scaling? Might want to check that long messages still wrap correctly.